ReactWindows/ReactNative.Shared/Views/View/ReactViewManager.cs

```diff
@@ -1,12 +1,20 @@
+using Newtonsoft.Json.Linq;
 using ReactNative.Reflection;
 using ReactNative.UIManager;
 using ReactNative.UIManager.Annotations;
+using ReactNative.UIManager.Events;
```

Comment:
Lots of new `using` statements but no new functionality.

Reply:
Fixed